packages/react/src/components/GroupedList/GroupedList.base.tsx

```diff
@@ -154,6 +161,7 @@ export class GroupedListBase extends React.Component<IGroupedListProps, IGrouped
         shouldEnterInnerZone={shouldEnterInnerZone}
         className={css(this._classNames.root, focusZoneProps.className)}
       >
+        <FocusRects />
```

Comment:
Does `GroupedListV2` need this too?

Reply:
I couldn't repro the issue in v2, though I'm not entirely sure why not. Some other component in the example might be initiating focus rects. I can add `useFocusRects()` in v2 just to be safe though 😅

Reply:
If it's not reproing let's leave it be. These components are complicated and I'd prefer to minimize changes to just the essentials.